src/reference/asciidoc/changes-5.5-6.0.adoc

```diff
@@ -0,0 +1,26 @@
+[[migration-5.5-6.0]]
+=== Changes between 5.5 and 6.0
+
+[[x6.0-new-components]]
+=== New Components
+TODO
+
+[[x6.0-general]]
+=== General Changes
+
+[[x6.0-poller]]
+==== @Poller
```

Comment:
This info has to be added to the `whats-new.adoc`. Because this is exactly what is new in this `6.0` version.
We will create a `changes-5.5-6.0.adoc` file, when we start `6.1` version.

The point is to let end-users to find easily what is new in the project just from a first page of our doc.